src/styles/components/_home.scss

```diff
@@ -43,6 +42,7 @@ $breakpoint-tablet: 820px;
   &-top__top-image {
     width: 100%;
     margin: 3.75rem 0; // top&bottom: 60px/16; left&right: 0
+    background-color: var(--background-color);
```

Comment:
I don't see a reason to put this here vs. where it was on line 9 before. I'd move it back so it applies to the whole section.

Reply:
Done! :)